tcms/testplans/tests/tests.py

```diff
@@ -74,7 +74,55 @@ def test_open_plans_search(self):
     def test_plan_new_get(self):
         location = reverse('plans-new')
         response = self.client.get(location, follow=True)
+
         self.assertEqual(response.status_code, HTTPStatus.OK)
+        self.assertContains(
+            response,
+            '<input class="bootstrap-switch" name="auto_to_plan_author" type="checkbox" '
+            'checked>', html=True)
+        self.assertContains(
+            response,
+            '<input class="bootstrap-switch" name="auto_to_case_owner" type="checkbox" '
+            'checked>', html=True)
+        self.assertContains(
+            response,
+            '<input class="bootstrap-switch" name="auto_to_case_default_tester" type="checkbox" '
+            'checked>', html=True)
+        self.assertContains(
+            response,
+            '<input class="bootstrap-switch" name="notify_on_plan_update" type="checkbox" '
+            'checked>', html=True)
+        self.assertContains(
+            response,
+            '<input class="bootstrap-switch" name="notify_on_case_update" type="checkbox" '
+            'checked>', html=True)
```

Comment:
Missing check for the is_active field.

TIP: to avoid copying the entire HTML you can also try to match the input name and the checked property. Not sure how well/easy that is to do. If it is not obvious in 5 minutes leave it like it is now.

Reply:
Unfortunately, the testing framework is not that flexible. Adding `html=True` allows me to switch the places of the HTML properties, but if I delete one, then in throws an AssertionError

Reply:
Still missing assertion for the is_active checkbox.

Reply:
Done